.github/workflows/pr-build.yml

```diff
@@ -68,12 +63,3 @@ jobs:
         run: |
           find . -type d -name "node_modules" -prune -o -type f -iregex '.*.ts\|.*.js\|.*.tsx\|.*.jsx' -exec ./build/set-mtime-to-md5.sh {} \;
           yarn run lint-check
-
-      - name: Build beta
-        run: yarn run build-beta
```

Comment:
The build is part of the PR check

Reply:
Yeah I am just playing around with stuff tbh.